docs/c-runtime-library/locale-names-languages-and-country-region-strings.md

```diff
@@ -37,6 +37,22 @@ _wsetlocale(LC_ALL, L"de-DE");
 _wsetlocale(LC_ALL, L"LC_MONETARY=en-GB;LC_TIME=es-ES");
 ```
 
+
+## UTF-8 Support
+
+Starting in Windows 10 build 17134 (April 2018 Update), the Universal C Runtime supports using a UTF-8 code page. This means that narrow (char) strings passed to C runtime functions will expect strings in the UTF-8 encoding and properly understand them. To enable this, use "UTF-8" as the code page when using setlocale. For example, [setlocale(LC_ALL, ".utf8")](reference/setlocale-wsetlocale.md) will use the current ACP for the locale and UTF-8 for the code page.
+
+After calling setlocale(LC_ALL, ".UTF8"), you may pass "😊" to mbtowcs and it will be properly translated to a wide string, whereas previously there was not a locale setting available to do this.
+
+UTF-8 mode is also enabled for functions that have historically translated narrow strings using the default Windows ANSI code page (ACP). For example, calling [_mkdir("😊")](reference/mkdir-wmkdir.md) while using a UTF-8 code page will correctly produce a directory with that emoji as the folder name, instead of requiring the ACP to be changed to UTF-8 prior to running your program. Likewise, calling [_getcwd()](reference/getcwd-wgetcwd.md) inside of that folder will return a UTF-8 encoded string. For behavior compatibility, the ACP is still used if the C locale code page is not set to UTF-8.
+
+There are some aspects of the C Runtime that are not able to use UTF-8 because they are set during program startup and must use the default Windows ANSI code page (ACP): [__argv](argc-argv-wargv.md), [_acmdln](acmdln-tcmdln-wcmdln.md), and [_pgmptr](pgmptr-wpgmptr.md).
```

Comment:
"such as:"

Reply:
I feel "such as" suggests it is a not the full list, no? I'll reword "There are some" -> "The following"?

Reply:
The sentence starts you thinking that it isn't the full list: "there are some aspects". Pick whichever matches what the situation is here. Are you showing some or all aspectes?